### ISSUE TITLE
[Feature Request] allow filtering

### ISSUE THREAD

**Issue by cracktorio** [tranxuanthang/lrcget#128]
Please allow filtering the type of song, for example by if it's instrumental, synced, plain or not found.
This would be very helpful for if songs aren't detected due to metadata mismatch, and then being able to find those problems without having to constantly scroll.

**Comment by phanion** [comment]
agreed this would be very useful since that is basically the logical next step in your workflow

**Comment by tranxuanthang** [comment]
I'll track this feature and notify new updates at #37. Allow me to close this.
